stock_packaging_calculator/tests/common.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo.tests import TransactionCase
+
+
+class TestCommon(TransactionCase):
+    at_install = False
+    post_install = True
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
FYI, inheriting from BaseCommon avoid adding this line : https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21